### DISCUSSION TITLE
How to change from staging to production?

### DISCUSSION THREAD

**Discussion by jmart1** [nabsul/kcert#90] [Q&A]
After testing in staging successfully, I went on deploy the helm install with Acme production URL set.

I executed this... 

`helm install kcert nabsul/kcert -n kcert --debug --set acmeTermsAccepted=true,acmeEmail=jxxxxx@gmail.com,acmeDirUrl=https://acme-v02.api.letsencrypt.org/directory`

and get this error....

```
install.go:194: [debug] Original chart version: ""
install.go:211: [debug] CHART PATH: /home/jmart/.cache/helm/repository/kcert-1.0.5.tgz

Error: INSTALLATION FAILED: cannot re-use a name that is still in use
helm.go:84: [debug] cannot re-use a name that is still in use
helm.sh/helm/v3/pkg/action.(*Install).availableName
        helm.sh/helm/v3/pkg/action/install.go:486
helm.sh/helm/v3/pkg/action.(*Install).RunWithContext
        helm.sh/helm/v3/pkg/action/install.go:200
main.runInstall
        helm.sh/helm/v3/cmd/helm/install.go:280
main.newInstallCmd.func2
        helm.sh/helm/v3/cmd/helm/install.go:139
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.6.1/command.go:916
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.6.1/command.go:1044
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.6.1/command.go:968
main.main
        helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
        runtime/proc.go:250
runtime.goexit
        runtime/asm_amd64.s:1571
INSTALLATION FAILED
main.newInstallCmd.func2
        helm.sh/helm/v3/cmd/helm/install.go:141
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.6.1/command.go:916
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.6.1/command.go:1044
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.6.1/command.go:968
main.main
        helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
        runtime/proc.go:250
runtime.goexit
        runtime/asm_amd64.s:1571
```

Is there something I should have done first before executing the helm command?

**Comment by jmart1** [comment]
I figured out the problem.  I used the wrong helm command.  I tried to delete this discussion post, but the delete option is not displaying.

**Comment by jmart1** [comment]
Actually, after switching to production, using the helm command, the old staging certificate is still being used.  All three browsers, plus curl, keep getting the staging cert.  Do I need to delete the old cert first?

**Reply by nabsul** [reply]
Yes, you'll need to delete the cert and recreate it. KCert only looks at the expiration date to decide whether a cert needs to be regenerated. 

Your other option is to enable the ShowRenewButton configuration on KCert. If you do that, a "renew" button will be shown from the tool's web interface.
